package.json

```diff
@@ -28,10 +28,10 @@
     "bson": "^1.1.1",
     "denque": "^1.4.1",
     "require_optional": "^1.0.1",
-    "safe-buffer": "^5.1.2",
-    "util.promisify": "^1.0.1"
+    "safe-buffer": "^5.1.2"
   },
   "devDependencies": {
+    "util.promisify": "^1.0.1",
```

Comment:
nit: there's a marginal effort to keep this list alphabetically sorted

Reply:
👍